libsigner/src/messages.rs

```diff
@@ -927,11 +998,70 @@ impl SignerMessage {
             },
             Self::BlockResponse(_) => BLOCK_SLOT_ID,
             Self::Transactions(_) => TRANSACTIONS_SLOT_ID,
+            Self::Nack(_) => NACK_SLOT_ID,
         };
         SIGNER_SLOTS_PER_USER * id + slot_id
     }
 }
 
+/// Signed network packets sent by signers along with their view
+/// of stacks tip consensus hash and block height for
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct PacketMetadata {
```

Comment:
Nit: calling this `PacketMetadata` is confusing, because it contains both metadata and a payload.  Maybe this could be called something more like `PacketMessage` in order to convey that it's a container for these two things?

Reply:
Sure, but `Packet` type holds a msg:Message so maybe that'll be repetitive too. 
I had it as `PacketInfo` before. Can rename to `PacketMessage` for now and we can alway refactor.